packages/test-utils/src/reporter.ts

```diff
@@ -200,6 +202,7 @@ class ReplayReporter<TRecordingMetadata extends UnstructuredMetadata = Unstructu
   upload = false;
   filter?: (r: RecordingEntry<TRecordingMetadata>) => boolean;
   recordingsToUpload: ExternalRecordingEntry<TRecordingMetadata>[] = [];
+  _testRunShardIdPromise: Promise<TestRunPendingWork> | null = null;
```

Comment:
Any reason not to just make this private?
```suggestion
  private testRunShardIdPromise: Promise<TestRunPendingWork> | null = null;
```

Reply:
Not making it private is an oversight on my part but I think it's still worth maintaining the `_` convention, even when using `private` keyword.

Reply:
Huh. Why?

Reply:
`private` doesn't hide it at a runtime level, just at the type level. I find it useful to know what's private and what's not when inspect values in the debugger

Reply:
That's fair. I guess it depends on what our compile target is. I was assuming TS would get compiled to `#testRunShardIdPromise` but I guess it doesn't?

Reply:
Ye, it doesn't do that - regardless of the target. To use private fields u have to use `#` syntax, not the `private` keyword

Reply:
TIL

Bad assumption on my part about TS :) Thanks for sharing